<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgNewOilChange</class>
 <widget class="QDialog" name="DlgNewOilChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Oil Change</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/fuel1.png</normaloff>:/img/fuel1.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Date</string>
       </property>
       <property name="buddy">
        <cstring>date</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="date"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Kms</string>
       </property>
       <property name="buddy">
        <cstring>spinKms</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Brand</string>
       </property>
       <property name="buddy">
        <cstring>txtBrand</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Where</string>
       </property>
       <property name="buddy">
        <cstring>txtWhere</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Quantity</string>
       </property>
       <property name="buddy">
        <cstring>txtQuantity</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtBrand"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtWhere"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtQuantity"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinKms"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>date</tabstop>
  <tabstop>spinKms</tabstop>
  <tabstop>txtBrand</tabstop>
  <tabstop>txtWhere</tabstop>
  <tabstop>txtQuantity</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgNewOilChange</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgNewOilChange</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
